components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -560,6 +532,25 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
     }
 }
 
+async fn async_checkpoint(
+    scheduler: &Scheduler<checkpoint::Task>,
+    parent_region: u64,
+    split_regions: Vec<u64>,
+    cur_suffix: u64,
+    log_index: u64,
+) {
+    let (tx, rx) = oneshot::channel();
```

Comment:
how about return the executing duration and log it in the code line 500?

Reply:
Done